src/System.Runtime.Extensions/tests/System/Convert.ToDouble.cs

```diff
@@ -103,15 +103,15 @@ public void FromString()
         }
 
         [Fact]
-        [SkipOnTargetFramework(~TargetFrameworkMonikers.NetFramework)]
+        [SkipOnTargetFramework(~TargetFrameworkMonikers.NetFramework | TargetFrameworkMonikers.UapAot)]
```

Comment:
I assume these are due to @tannergooding's recent changes?

Reply:
Why are just these tests being skipped? We had a lot more deltas on the formatting changes...